### ISSUE TITLE
data needed to run example

### ISSUE THREAD

**Issue by thisIsMikeKane** [ecobee/building-controls-simulator#133]
I have built the latest version of BCS from source using `docker up` on Windows. However, I'm getting an error when I try to run `demo_LocalSource.ipynb`

![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/997703/105567050-91f27780-5cfd-11eb-9349-c1855d4e9047.png)
    
It seems the actually directory specified by `LOCAL_CACHE_DIR` (i.e., `/home/bcs/lib/building-controls-simulator/data/` ) does not exist. 

I haven't specified any of the cloud credentials, so it should be trying to do everything locally. But, do I need to copy local files to a specific directory? If so, what files do I need to copy to the local data directory?

I resolved the issue by copying 2756663bc57afe940c552ab1080f2ec09ef54e24.csv.zip to /home/bcs/lib/building-controls-simulator/data/input/local/ creating the necessary parent directories. 

tldr; The need to copy the example `.csv.zip` to the `data/input/local/` should be documented as an step when installing BCS. Alternatively, could this file be included in the repo? (Probably not given the NDA on the DYD data.)

**Comment by tomstesco** [comment]
Yes the documentation needs to be streamlined - I tried to cover this in the section `Using locally cached data`. I added an extra line explaining what the contents of the data files are for context, I don't think that helps you but if someone is unfamiliar with DYD.

**Comment by thisIsMikeKane** [comment]
I just provided a pull request that provides clarification in `notebooks\demo_LocalSource.ipynb` since users making new scripts would likely start from one of the demos. #137

**Comment by parthmaheshwari** [comment]
I faced this error as well, where did you get the 2756663bc57afe940c552ab1080f2ec09ef54e24.csv.zip file from? I don't see it in the repository. Do we get this file after signing up for DYD program?

**Comment by tomstesco** [comment]
To make the setup easier we're working on a dummy data generator function so we wont need to store data in the repository or require that you have access to DYD or other data sources. That should be ready in `master` within the week.
